instance/tests/models/test_openedx_instance.py

```diff
@@ -467,6 +467,10 @@ def test_get_load_balancer_configuration(self, mocks):
             backend_map, config, domain_names, instance.active_appserver.server.public_ip
         )
 
+        # Test configuration after terminating appserver
+        with patch('instance.openstack.get_server_public_address', return_value=None):
+            self.assertEqual(instance.get_load_balancer_configuration(), ([], []))
```

Comment:
@antoviaque I'm still waiting for your answer here.  I personally wouldn't make that change, since I don't think the tests are simply concatenated, but if you prefer, I'm happy to change it.